.github/workflows/main.yml

```diff
@@ -71,6 +71,7 @@ jobs:
         uses: chromaui/action@v1
         with:
```

Comment:
Looks like missed a spot in nimble-components/src/table/tests/table-action-menu-opened-matrix.stories.ts
still references all-components